src/mem.cpp

```diff
@@ -142,12 +144,17 @@ static int gds_map_gdr_memory(gds_mem_desc_t *desc, CUdeviceptr d_buf, size_t si
 static int gds_unmap_gdr_memory(gds_mem_desc_t *desc)
 {
         int ret = 0;
+        #if defined(GDR_API_MAJOR_VERSION) && (GDR_API_MAJOR_VERSION > 1)
```

Comment:
can you create an helper function and use it both here and in gds_map_gdr_memory ?